tests/bluetooth/tester/src/btp/btp.h

```diff
@@ -75,8 +76,9 @@
 #define BTP_SERVICE_ID_TBS	27
 #define BTP_SERVICE_ID_TMAP	28
 #define BTP_SERVICE_ID_OTS	29
+#define BTP_SERVICE_ID_BAS	31
```

Comment:
30 is PBP, right?

Just be aware that we are modifying this to be hex in https://github.com/zephyrproject-rtos/zephyr/pull/80740

Reply:
Correct. 

I see, I will wait for that PR to be merged.

Reply:
I would be OK to also add the PBP service value here, so that it's more obvious why 31 is used for BAS.